fattmerchant-ios-sdk/Cardpresent/AnywhereCommerce/AWCDriver.swift

```diff
@@ -62,9 +62,9 @@ class AWCDriver: NSObject, MobileReaderDriver, CBCentralManagerDelegate {
   ///   - completion: a block to run once initialization is complete. The block will receive the value 'true' if initialization was successful, false otherwise
   func initialize(args: [String: Any], completion: @escaping (Bool) -> Void) {
     guard
-      let merchant = args["merchant"] as? Merchant,
-      let worldnetSecret = merchant.emvTerminalSecret(),
-      let worldnetTerminalId = merchant.emvTerminalId()
+      let awcDetails = args["awc"] as? AWCDetails,
+      let worldnetSecret = awcDetails.terminal_secret,
+      let worldnetTerminalId = awcDetails.terminal_id
```

Comment:
`AWCDetails#terminal_id` and `AWCDetails.terminal_secret` are not optional, so this guard throws a compile error